module/zfs/vdev_removal.c

```diff
@@ -1363,6 +1368,20 @@ spa_vdev_copy_impl(vdev_t *vd, spa_vdev_removal_t *svr, vdev_copy_arg_t *vca,
 	range_tree_destroy(segs);
 }
 
+/*
+ * The size of each removal mapping is limited by the tunable
+ * zfs_remove_max_segment, but we must adjust this to be a multiple of the
+ * pool's ashift, so that we don't try to split individual sectors regardless
+ * of the tunable value.  (Note that device removal requires that all devices
+ * have the same ashift, so there's no difference between spa_min_ashift and
+ * spa_max_ashift.) The raw tunable should not be used elsewhere.
+ */
+uint64_t
+spa_remove_max_segment(spa_t *spa)
+{
+	return (P2ROUNDUP(zfs_remove_max_segment, 1 << spa->spa_max_ashift));
```

Comment:
It appears `zfs_remove_max_segment` was never documented in `zfs-module-parameters(5)`, nor was `vdev_removal_max_span` or `zfs_removal_suspend_progress`.  Now would be a good time to do so.

Reply:
vdev_removal_max_span is already documented.  I added the other two.